### PR TITLE
Propose adding link to Fission setup guide

### PR DESCRIPTION
**Pull request by bmann**
hi all! We've got instructions on our guide on how to use a Cloudflare hosted DNS with the gateway and serve up an IPNS site from Fission. LMK if this looks OK.

### CODE REVIEW

**Review by OhArthits — APPROVED** [review]
การทำงานเป็นทีม

**Comment by thibmeu** [comment]
Hi @bmann, Thanks for the PR.
It looks good for me.

**Review by thibmeu — APPROVED** [review]

File: products/distributed-web/src/content/ipfs-gateway/automated-deployment.md [hunk 1]
```diff
@@ -24,3 +24,6 @@ hash. There are several tools that help with different parts of this:
 - [dnslink-cloudflare](https://github.com/ipfs-shipyard/dnslink-cloudflare) is a
   script to programmatically update DNSLink records. This can be run with the
   `-Q` flag of `ipfs add` that only outputs the top-level hash.
+- [fission ipns support](https://guide.fission.codes/developers/custom-domains/using-cloudflare-ipfs-gateway)
```
Comment (pedrosousa) [inline products/distributed-web/src/content/ipfs-gateway/automated-deployment.md]:
```suggestion
- [Fission's IPNS support](https://guide.fission.codes/developers/custom-domains/using-cloudflare-ipfs-gateway) 
```

File: products/distributed-web/src/content/ipfs-gateway/automated-deployment.md [hunk 2]
```diff
@@ -24,3 +24,6 @@ hash. There are several tools that help with different parts of this:
 - [dnslink-cloudflare](https://github.com/ipfs-shipyard/dnslink-cloudflare) is a
   script to programmatically update DNSLink records. This can be run with the
   `-Q` flag of `ipfs add` that only outputs the top-level hash.
+- [fission ipns support](https://guide.fission.codes/developers/custom-domains/using-cloudflare-ipfs-gateway) 
+  lets you use the the Fission IPFS app publishing system from the CLI
```
Comment (pedrosousa) [inline products/distributed-web/src/content/ipfs-gateway/automated-deployment.md]:
```suggestion
  lets you use the Fission IPFS app publishing system from the CLI
```

File: products/distributed-web/src/content/ipfs-gateway/automated-deployment.md [hunk 3]
```diff
@@ -24,3 +24,6 @@ hash. There are several tools that help with different parts of this:
 - [dnslink-cloudflare](https://github.com/ipfs-shipyard/dnslink-cloudflare) is a
   script to programmatically update DNSLink records. This can be run with the
   `-Q` flag of `ipfs add` that only outputs the top-level hash.
+- [fission ipns support](https://guide.fission.codes/developers/custom-domains/using-cloudflare-ipfs-gateway) 
+  lets you use the the Fission IPFS app publishing system from the CLI  
+  or Github Actions, while using Cloudflare-managed DNS and gateway
```
Comment (pedrosousa) [inline products/distributed-web/src/content/ipfs-gateway/automated-deployment.md]:
```suggestion
  or from GitHub Actions, while using Cloudflare-managed DNS and gateway.
```

**Review by lukevalenta — APPROVED** [review]

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**